platform/linux-generic/odp_schedule_basic.c

```diff
@@ -221,11 +226,41 @@ static __thread sched_local_t sched_local;
 /* Function prototypes */
 static inline void schedule_release_context(void);
 
+static int read_config_file(sched_global_t *sched)
+{
+	const char *str;
+	int val = 0;
+
+	ODP_PRINT("Scheduler config:\n");
+
+	str = "sched_basic.prio_spread";
+	if (!_odp_libconfig_lookup_int(str, &val)) {
+		ODP_ERR("Config option '%s' not found.\n", str);
```

Comment:
Same comments here about requiring fields to be present. We should use built-in defaults if they are not.

Reply:
_odp_libconfig_lookup_int() first checks user provided config file and falls back to the default config file. There's always a config file with this option set.